src/content/1.7/development/configuration.md

```diff
@@ -0,0 +1,382 @@
+---
+title: PrestaShop configuration data storage
+weight: 80
+---
+
+# PrestaShop configuration data storage
+
+## PrestaShop Configuration object
+
+PrestaShop provides a configuration system to store values based on a key-values data storage.
+
+You can access and modify these data using class `\Configuration`. It is able to store and fetch key-values data and persists it into `ps_configuration` SQL table.
+
+Multiple methods are available in the class for you to browser and modify, the most popular are:
+
+- `\Configuration::get($key, $idLang = null, $idShopGroup = null, $idShop = null, $default = false)`
+- `\Configuration::hasKey($key, $idLang = null, $idShopGroup = null, $idShop = null)`
+- `\Configuration::set($key, $values, $idShopGroup = null, $idShop = null)`
+
+## Standard configuration values
+
+Most configuration settings of PrestaShop are stored using this system: whether or not SSL is enabled, the shop URL, whether or not Taxes are enabled, whether or not to accept out-of-stock orders ... here is the list of the standard configuration keys you can encounter:
```

Comment:
I don't think we can call it "list of the standard configuration keys", since a lot of keys are related to modules. :thinking: